test/aws.test.js

```diff
@@ -0,0 +1,100 @@
+/* global describe it before after */
+
+const assert = require('assert')
+const kmsHelper = require('../lib/kms-helper')
+const AWS = require('aws-sdk-mock')
+const sinon = require('sinon')
+const lambdaTester = require('lambda-tester')
+const handler = require('../index.js').handler
+const event = require('../event.json')
+const KinesisWriter = require('../lib/KinesisWriter')
+
+const getRDSString = (file) => {
+  return require('dotenv').config({ path: `${file}` }).parsed['DISCOVERY_STORE_CONNECTION_URI']
+}
+
+describe('AWS', function () {
+  this.timeout(5000)
+  require('dotenv').config({ path: './.env' })
+
+  describe('kmsHelper util', () => {
+    it('should fail when there is no encrypted key', () => {
+      process.env['DISCOVERY_STORE_CONNECTION_URI'] = ''
+      try {
+        kmsHelper.decryptDbCreds()
+      } catch (e) {
+        assert.equal(e.message, 'Missing DISCOVERY_STORE_CONNECTION_URI env variable; aborting.')
+      }
+    })
+
+    it('should call decrypt the db uri', () => {
+      process.env['DISCOVERY_STORE_CONNECTION_URI'] = getRDSString('./test.deploy.env')
```

Comment:
Yeah interesting challenge. I submitted a question to lambda-tester:

https://github.com/vandium-io/lambda-tester/issues/16